.github/workflows/integration-tests.yml

```
@@ -0,0 +1,33 @@
+name: Integration Tests
+on:
+  merge_group:
+    types: [checks_requested]
+  pull_request:
+
+jobs:
+  test-basic-flow:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Build and push
+        uses: docker/bake-action@v3
+        with:
+          files: docker-compose.yml
+          push: false
+      - name: Seed database
+        run: docker compose run elixir /bin/sh -c "cd apps/domain && mix do ecto.reset, ecto.seed"
```

DB should be uninitialized at this point, so you can probably drop the `ecto.reset`.

```suggestion
        run: docker compose run elixir /bin/sh -c "cd apps/domain && mix ecto.seed"
```